app/assets/javascripts/Models.elm

```diff
@@ -174,6 +182,12 @@ path base params =
                 , params.latLng
                     |> Maybe.map (\( lat, lng ) -> [ ( "lat", toString lat ), ( "lng", toString lng ) ])
                     |> Maybe.withDefault []
+                , params.fType
```

Comment:
This is a perfect fit for the new and shiny `SelectList` api. You can do something like this:

```elm
        queryParams =
            select <|
                List.map maybe
                    [ Maybe.map (\q -> ( "q", q )) params.q
                    , Maybe.map (\s -> ( "s", toString s )) params.s
                    , Maybe.map (\l -> ( "l", toString l )) params.l
                    , Maybe.map (\( lat, _ ) -> ( "lat", toString lat )) params.latLng
                    , Maybe.map (\( _, lng ) -> ( "lng", toString lng )) params.latLng
                    , Maybe.map (\fType -> ( "fType", toString fType )) params.fType
                    , Maybe.map (\fName -> ( "fName", fName )) params.fName
                    ]
```